lib/Header/Header.less

```diff
@@ -182,4 +182,12 @@
 			height: 159px;
 		}
 	}
+}
+
+.hidden {
+  display: none!important;
+}
+
+.moon-input-header .moon-input-header-input-decorator .moon-input.header-marquee {
+  line-height: 1.24em;
```

Comment:
Where did this number come from? I noticed that when switching between the Input and the Marquee, there was a noticeable shift in the positioning of the controls.